### PR TITLE
Add procs to get real and imaginary part of complex numbers

### PR DESCRIPTION
**Pull request by Paalon**

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -65,6 +65,8 @@
 - Added cotangent, secant and cosecant procs ``math.cot``, ``math.sec`` and ``math.csc``; and their hyperbolic, inverse and inverse hyperbolic functions, ``math.coth``, ``math.sech``, ``math.csch``, ``math.arccot``, ``math.arcsec``, ``math.arccsc``, ``math.arccoth``, ``math.arcsech`` and ``math.arccsch`` procs.
 - Added the procs ``math.floorMod`` and ``math.floorDiv`` for floor based integer division.
 - Added the procs ``rationals.`div```, ``rationals.`mod```, ``rationals.floorDiv`` and ``rationals.floorMod`` for rationals.
+- Added the procs ``rationals.real`` and ``ratioals.imag`` for returning real
```
Comment (NeeleshChandola) [inline changelog.md]: typo:  ratioals => rationals

Comment (NeeleshChandola) [inline changelog.md]: typo: ratioals => rationals

**Comment by andreaferretti** [comment]
I don't see anything wrong with `im` and `re`

**Comment by Paalon** [comment]
@Varriount We can not access via proc like this:
```nim
import complex
let a = im
echo re(a)
```

```bash
test.nim(3, 6) Error: undeclared identifier: 're'
```

**Comment by andreaferretti** [comment]
Well, then the fields just need to be made public

**Comment by Paalon** [comment]
I can't find how to make public the tupled parameters.

**Comment by andreaferretti** [comment]
Sorry, they are already public. You just need to access them via `a.re` instead of `re(a)`

**Comment by Araq** [comment]
Why do we need these?

**Comment by dom96** [comment]
Yeah, I don't think this is needed. You just have to access the fields like all fields are accessed.

**Pull request closed without merging**